sample/src/test/java/io/ona/kujaku/sample/activities/MainActivityTest.java

```diff
@@ -0,0 +1,106 @@
+package io.ona.kujaku.sample.activities;
```

Comment:
Since these are instrumented tests, don't you think they should be in `sample/src/androidTest` instead of `sample/src/test`

Reply:
Nope, they are not instrumented.